### ISSUE TITLE
clientupdate.curse.com down

### ISSUE THREAD

**Issue by khogeland** [arnauddupuis/wocli#13]
It looks like the host that was previously serving the database file wocli is using no longer exists (cloudflare times out while connecting).

http://clientupdate.curse.com/feed/Complete.xml.bz2

**Comment by arnauddupuis** [comment]
Yes I know it's due to Curse shifting from the XML data transmission to JSON. 
I have started to work on that but it's not immediate. New DB URL is https://clientupdate-v6.cursecdn.com/feed/addons/432/v10/complete.json.bz2.
